scripts/proto_codegen_jaeger.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+#
+# Regenerate python code from Jaeger protos in
+# https://github.com/jaegertracing/jaeger-idl
+#
```

Comment:
What's this file for?

Reply:
Submitted by accident. It's a script to regenerate jaeger protos. not needed for this PR.